www/src/pages/components/navigation-bar/ios/index.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Navigation Bar
+description: Navigation controls displayed along the top of the screen.
+---
+
+import { graphql, Link } from 'gatsby';
+import {
+    ComponentHeader,
+    ComponentFooter,
+    DeprecatedComponentAlert,
+} from 'components/thumbprint-components';
+
+<ComponentHeader data={props.data} />
+
+## Summary
+
+Navigation bars for iOS are implemented using the built-in UIKit component `UINavigationBar`,
+however they should be configured using the Thumbprint decorator method `NavigationBar.configure(_:appearance:content:)`
+to apply one of the standard Thumbprint navigation bar styles.
+
+### Appearance
+
+There are three standard navigation bar appearances. In addition, custom themes can be defined by calling
+the `NavigationBar.Appearance` constructor directly.
+
+#### Default
+
+![Default navigation bar](/img/ios-navigation-bar-default.png)
+
+#### Shadowless
+
+![Shadowless navigation bar](/img/ios-navigation-bar-shadowless.png)
+
+#### Transparent
+
+![Transparent navigation bar](/img/ios-navigation-bar-transparent.png)
+
+### Content Style
```

Comment:
Are these associated with specific appearances of do we mix and match? Feels weird to keep them as separate types/parameters if they are going to be tied up to particular appearances.

Reply:
We mix and match- most commonly (/only) with the `.transparent` appearance, where we might use either content style depending on whether the content underneath is light or dark

Reply:
Ok. Still think it might make more sense to make the content style part of the theme, we'd only end up adding one prebuilt type after all (transparent appearance becomes transparent/light and transparent/dark).

No need to change it now and besides we probably will need to have this conversation again if we ever go for dark mode, which I'd want to tackle at some point for AX reasons mostly.